apps/menu-bar/modules/menu-bar/ios/MenuBarExceptions.swift

```diff
@@ -0,0 +1,19 @@
+import ExpoModulesCore
+
+internal class CLIOutputError: GenericException<String> {
+  override var reason: String {
+    "Output error: \(param)"
```

Comment:
The `runCli` function expects an error message containing a stringified JSON that we can parse to reconstruct the error thrown by the CLI in react-native, that's why we use the `---- thrown error ----` structure. 

https://github.com/expo/orbit/blob/c185c4221e6c8c9d0fa0a910a19bca45aedd59cc/apps/menu-bar/modules/menu-bar/index.ts#L24


```suggestion
    "\(param)"
```

Reply:
Ah, I see. Cool I'll get that fixed